raster/r.sunmask/r.sunmask.html

```diff
@@ -1,34 +1,58 @@
 <h2>DESCRIPTION</h2>
 
 <em>r.sunmask</em> creates an output map layer based on an input elevation
-raster map layer and the sun position. The output map layer contains the
-cast shadow areas arising from sun shine and elevations. The user can define
-the sun position either directly or the module calculates it from given
-location and date/time parameters using the
-<a href="http://rredc.nrel.gov/">NREL</a> sun position algorithm. So either
-"A:"-parameters to specify the exact known sun position or "B:-parameters"
-to specify date/time for sun position calculation by <em>r.sunmask</em> itself
-have to be used.
-
-<p>The module performs sunset/sunrise checks and refraction correction for sun
-position calculation. Local coordinate systems are internally transformed to
-latitude/longitude for the SOLPOS algorithm. The elevation is not considered
-in the sunset/sunrise calculations.
+raster map layer and the position of the sun. The output map layer contains the
+cast shadow areas resulting from sunlight and elevation. The user can either
+specify the sun position directly or the module calculates it from given
+location and date/time parameters using the SOLPOS (Solar and Moon Position Algorithm)
+developed by the
+<a href="https://www.nrel.gov/">National Renewable Energy Laboratory</a>
+(NREL). SOLPOS operates in two modes, either
+
+<ul>
+<li>(A) parameters to specify the exact known position of the sun, or</li>
+<li>(B) parameters to specify the date/time for the sun position calculation
+     by <em>r.sunmask</em> itself</li>
+</ul>
+must be used.
+
+<p>
+The module performs sunset/sunrise checks and refraction correction for
+sun position calculation. Local coordinate systems are internally transformed
+to latitude/longitude for the SOLPOS algorithm. Elevation is not taken into
+account for sunset/sunrise calculations.
+
+<p>
+The solar zenith angle ("sun angle above horizont") is defined as the angle
+between the horizont and the vertical (directly overhead or zenith). Its
+values can range from 90&deg;, when the sun is directly overhead, to 0&deg;,
+when the sun is on the horizon. Values lower than 0&deg; indicate that the
+sun is below the horizon.
+
+<p>
+The solar azimuth angle ("sun azimuth") defines the direction of the sun.
+It is the angle between north (in the northern hemisphere) or south (in
+the southern hemisphere) and the projection of the sun's rays onto the
```

Comment:
Not sure if the angle is calculated from south in the southern hemisphere because below it says "Specifically, an azimuth of 180&deg; means the sun is in the south"

Reply:
Right, the hemisphere might not matter in the computational context. I will remove that part.